packages/tempus-client_v3/src/services/NotificationService.spec.ts

```diff
@@ -0,0 +1,308 @@
+      mockGet.mockReturnValue(mockStoredNotifications);
+      notificationService = new NotificationService(mockStorageService);
+    });
+
+    afterAll(() => {
+      mockGet.mockReturnValue([]);
+    });
+
+    it('emits last stored notifications', () => {
+      expect(mockListener).not.toHaveBeenCalled();
+
+      notificationService.getLastItems().subscribe(mockListener);
+
+      expect(mockListener).not.toHaveBeenCalled();
+
+      act(() => {
+        jest.advanceTimersByTime(100);
+      });
+
+      expect(mockDelete).toHaveBeenCalledTimes(1);
+
+      expect(mockListener).toHaveBeenNthCalledWith(1, {
+        category: 'Transaction',
+        chain: 'ethereum',
+        content: 'content 3',
+        dismissed: false,
+        id: 'stored003',
+        level: 'warning',
+        timestamp: 1652569200010,
+        title: 'StoredNotification 3',
+      });
+
+      expect(mockListener).toHaveBeenNthCalledWith(2, {
+        category: 'Transaction',
+        chain: 'ethereum',
+        content: 'content 2',
+        dismissed: false,
+        id: 'stored002',
+        level: 'warning',
+        timestamp: 1652569200001,
+        title: 'StoredNotification 2',
+      });
+
+      expect(mockListener).toHaveBeenNthCalledWith(3, {
+        category: 'Transaction',
+        chain: 'fantom',
+        content: 'content 1',
+        dismissed: false,
+        id: 'stored001',
+        level: 'info',
+        link: 'someLink',
+        linkText: 'someLinkText',
+        timestamp: 1652569200000,
+        title: 'StoredNotification 1',
+      });
+    });
+  });
+
+  describe('deleteNotifications()', () => {
+    beforeEach(() => {
+      notificationService = new NotificationService(mockStorageService);
+    });
+
+    it('deletes all notifications', () => {
+      notificationService.deleteNotifications();
+
+      expect(mockDelete).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe('dismissNotification()', () => {
+    beforeEach(() => {
+      mockGet.mockReturnValue(mockStoredNotifications);
+      notificationService = new NotificationService(mockStorageService);
+    });
+
+    it('dismisses a notification', () => {
+      notificationService.dismissNotification('stored002');
+
+      expect(mockSet).toHaveBeenCalledTimes(1);
+      expect(mockSet).toBeCalledWith(NOTIFICATIONS_KEY, [
+        {
+          category: 'Transaction',
+          chain: 'fantom',
+          content: 'content 1',
+          dismissed: false,
+          id: 'stored001',
+          level: 'info',
+          link: 'someLink',
+          linkText: 'someLinkText',
+          timestamp: 1652569200000,
+          title: 'StoredNotification 1',
+        },
+        {
+          category: 'Transaction',
+          chain: 'ethereum',
+          content: 'content 2',
+          dismissed: true, // this changed
+          id: 'stored002',
+          level: 'warning',
+          timestamp: 1652569200001,
+          title: 'StoredNotification 2',
+        },
```

Comment:
Even in this case it is as simple as 
```suggestion
        {
          ...notification2,
          dismissed: true, // this changed
        },
```

Reply:
refactored